graylog2-server/src/main/java/org/graylog2/filters/StaticFieldFilter.java

```diff
@@ -70,24 +84,38 @@ public boolean filter(Message msg) {
         return false;
     }
 
-    private List<Map.Entry<String, String>> loadStaticFields(final String inputId) {
+    @Subscribe
+    public void handleInputCreate(final InputCreated event) {
+        LOG.debug("Load static fields for input <{}>", event.id());
+        scheduler.schedule(() -> loadStaticFields(event.id()), 0, TimeUnit.SECONDS);
+    }
+
+    @Subscribe
+    public void handleInputDelete(final InputDeleted event) {
```

Comment:
Adding `@SuppressWarnings("unused")` would disable the compiler warnings (and make the IDE happy).